src/services/swap_service.ts

```diff
@@ -749,7 +749,7 @@ export class SwapService implements ISwapService {
             isToETH,
             isMetaTransaction,
             shouldSellEntireBalance,
-            affiliateFees,
+            buyTokenAffiliateFees: affiliateFees,
```

Comment:
Also need to pass in `sellTokenAffiliateFees`. Maybe we can just  split the `affiliateFees` in `getSwapQuotePartialTransaction` to `sellTokenAffiliateFees` and `sellTokenAffiliateFees`